types/openstack-wrapper/index.d.ts

```diff
@@ -0,0 +1,274 @@
+// Type definitions for openstack-wrapper 2.1.6
+// Project: https://www.npmjs.com/package/openstack-wrapper
+// Definitions by: Sanjay Madane <https://github.com/sanjaymadane>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+declare module "openstack-wrapper" { 
```

Comment:
do not wrap in a module declaration, i.e. `declare module "openstack-wrapper" { ...`. instead, export declarations directelly:

```ts
 export interface RequestOption {
...
```